{
  "id": "CVE-2017-14503",
  "published": "2017-09-17T18:29:00",
  "modified": "2025-09-30T06:56:54.259130",
  "details": "libarchive 3.3.2 suffers from an out-of-bounds read within lha_read_data_none() in archive_read_support_format_lha.c when extracting a specially crafted lha archive, related to lha_crc16.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
    }
  ],
  "repo_url": "https://github.com/libarchive/libarchive",
  "cwes": [],
  "commits": [
    "19f23e191f9d3e1dd2a518735046100419965804"
  ]
}
